src/components/impact/styledImpact.js

```diff
@@ -1,28 +1,92 @@
 import styled from "styled-components";
+import { Highlight } from "../sharedComponents/sharedComponents";
 
 export const ImpactWrapper = styled.section`
     background: #F4F4F4;
     padding: 120px 0;
     display: flex;
     flex-direction: column;
+    z-index: 0;
 `
 export const BoxWrapper = styled.div`
     display: flex;
+    position: relative;
     width: 94.44%;
     justify-content: space-between;
     margin: 60px auto 120px auto;
+
+    @media (max-width: 600px) {
+        flex-direction: column;
+    }
+`
+export const BoxContain = styled.div`
```

Comment:
the dimensions of the box are off at 1024. It looks like the left and right margin outside the boxWrapper might be a little high or the width is off. Another place to look might be the space between the 2 boxContainers, it looks too big